UI/src/view/workListView.js

```diff
@@ -79,6 +79,14 @@
           .append(tableForGroupingValues('Ei tiedossa', workListItems.Unknown, 0));
     };
 
+    var addSpinner = function () {
+      $('#work-list').append('<div class="spinner-overlay modal-overlay"><div class="spinner"></div></div>');
```

Comment:
Why not palce this functions 'addSpinner' and 'removeSpinner' in workListView.js, since this file inherits from workListView.js and we just can call this functions